src/imitation/util/util.py

```diff
@@ -49,15 +49,30 @@ def make_vec_env(env_name: str,
       max_episode_steps: If specified, wraps VecEnv in TimeLimit wrapper with
           this episode length before returning.
```

Comment:
```suggestion
          this episode length before returning. Otherwise, defaults to `max_episode_steps` for `env_name` in the Gym registry.
```

Reply:
I expanded this comment a bit more in https://github.com/HumanCompatibleAI/imitation/pull/160/commits/365c1772e790e1956cf6d08c765ccda6a2a3d1cd. Wanted to note that the gym registry total timesteps thing is default behavior for `gym.make`.